README.md

```diff
@@ -6,24 +6,24 @@
    CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) and
    [Golang](https://golang.org/dl) installed.
 
-1. Check out the codebase
+2. Check out the codebase
```

Comment:
Please revert this, markdown does the right thing for you.

Reply:
TIL. thanks